docs/node-operators/index.mdx

```diff
@@ -22,6 +22,6 @@ The Mina nodes fulfill different roles within the network, including:
 
 1. [Block producers](/node-operators/block-producers) - this is analogous to being a Bitcoin "miner" or a "validator" in other proof-of-stake networks. By staking MINA, you can be selected to produce a block and win the block reward.
 
-2. SNARK coordinators - distribute work to a series of [SNARK workers](/node-operators/snark-workers) in parallel. 
+2. SNARK coordinators - distribute work to a series of [SNARK workers](/node-operators/snark-workers) in parallel. The SNARK worker nodes generate proofs of transactions. The block producer buys these proofs on the network (a "snarketplace"). For their efforts, the SNARK worker gets rewarded a part of the block reward.
```

Comment:
NIT: SNARK workers are not nodes. Also, saying the SNARK worker gets the rewards is a bit wrong -- it's the SNARK coordinator that determines the payout. SNARK workers are just stateless processes that do work for coordinators, we should minimize mentioning them here.

We should probably redo the names of this later, because the naming convention is unnecessarily confusing.